packages/simcore-sdk/src/simcore_sdk/node_ports_common/filemanager.py

```diff
@@ -349,7 +350,11 @@ async def upload_file(
                 upload_links,
                 uploaded_parts,
             )
-        except (r_clone.RCloneFailedError, exceptions.S3TransferError) as exc:
+        except (
+            CancelledError,
+            r_clone.RCloneFailedError,

```

Comment:
Once you have handled CancelledError and aborted upload, don't you want to reraise CancelledError??
What would happen if a user of this library uses ``upload_file`` inside a task that somehow gets canceled?